app/components/collections/works_component.html.erb

```diff
@@ -28,10 +28,21 @@
       <td><%= render Works::StateDisplayComponent.new(work: work) %></td>
       <td><%= I18n.l(work.updated_at.to_date, format: :abbr_month) %></td>
       <td><%= work.attached_files.count %></td>
-      <td><%= work.attached_files.sum(&:byte_size) %></td>
+      <td><%= number_to_human_size(work.attached_files.sum(&:byte_size)) %></td>
       <td><%= link_to work.purl, work.purl if work.purl %></td>
       <td><span class="fas fa-quote-left"></span> Cite</td>
     </tr>
   <% end %>
 </tbody>
 </table>
+
+<script>
+  var table = new simpleDatatables.DataTable("table", {
+    columns: [
+        // Sort the first column in ascending order
+        { select: 0, sort: "asc" },
+
+        // Disable sorting on second, and 4th - 7th columns
```

Comment:
I'm confused about this comment and how it relates to the next line.

Reply:
`{ select: [...], sortable: false }` is the config to disable sorting on the columns in the select. Happy to drop the comment if it's confusing/misleading.